tools/rawtools/rawtools.xml

```diff
@@ -0,0 +1,274 @@
+        </collection>
+
+        <!-- Metrics file -->
+
+        <collection name="output_metrics" type="list" label="${tool.name} - ${on_string}: Metrics" >
+            <filter>pq_options['metrics_boolean'] is True</filter>
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)\.raw_Metrics.txt" ext="txt" directory="output"/>
+        </collection>
+    </outputs>
+    <tests>
+
+        <!-- Test Matrix -->
+        <test>
+            <param name="input_raws" value="really_small.raw" ftype="thermo.raw"/>
+            <param name="pq_conditional|matrix_select" value="yes"/>
+            <output_collection name="output_matrix" type="list">
+                <element name="really_small" file="really_small.raw_Matrix.txt" ftype="txt" compare="sim_size" delta="3000"/>
+            </output_collection>
+        </test>
+
+        <!-- Test mgf conversion -->
+        <test>
+            <param name="input_raws" value="really_small.raw" ftype="thermo.raw"/>
+            <param name="pq_conditional|matrix_select" value="no"/>
+            <param name="mgf_conditional|mgf_select" value="yes"/>
+            <param name="mgf_conditional|mgf_refinement" value="yes"/>
+            <output_collection name="output_mgf" type="list">
+                <element name="really_small.raw" file="really_small.raw.mgf" ftype="mgf">
+                    <assert_contents>
+                        <has_text text="PEPMASS=399.347595214844"/>
+                        <has_text text="RTINSECONDS=487.3359"/>
+                        <has_text text="SCANS=18"/>
+                        <has_text text="101.29284 3.8678"/>
+                    </assert_contents>
+                </element>
+            </output_collection>
+        </test>
+
+        <!-- Test chromatogram -->
+        <test>
+            <param name="input_raws" value="really_small.raw" ftype="thermo.raw"/>
+            <param name="pq_conditional|matrix_select" value="no"/>
+            <param name="chromatogram_conditional|chromatogram_select" value="yes"/>
+            <output_collection name="output_chromatogram" type="list" >
+                <element name="really_small" file="really_small.raw_Ms_BP_chromatogram.txt" ftype="txt" compare="sim_size" delta="3000"/>
+            </output_collection>
+        </test>
+
+
+    </tests>
+    <help>
+**What it does**
+
+RawTools is an open-source and freely available package designed to perform scan data parsing and quantification, and quality control analysis (to be developed in further versions) of Thermo Orbitrap raw mass spectrometer files.
+
+RawTools is written in C# and uses the Thermo RawFileReader library. RawTools is the replacement for RawQuant Python package.
+
+https://github.com/kevinkovalchik/RawTools
+
+**License**
+
+Raw Tools is a free open-source project, following an Apache License 2.0.
+
+RawFileReader reading tool. Copyright © 2016 by Thermo Fisher Scientific, Inc. All rights reserved.
```

Comment:
Shit, if this is true we need to include a button as @chambm did for msconvert :(

Reply:
Well it is not a big deal, I have included it :-)